frontend/src/concepts/modelCatalog/types.ts

```diff
@@ -39,3 +42,18 @@ export type ModelCatalogSource = {
   source: string;
   models: CatalogModel[];
 };
+
+// Temporary type for MVP - will be replaced with remote model catalog sources
+// See: https://issues.redhat.com/browse/RHOAISTRAT-455
+export type ModelCatalogConfigMap = K8sResourceCommon & {
+  metadata: {
+    name: string;
+    labels: DashboardLabels & {
+      'opendatahub.io/model-catalog': 'true';
+    };
+  };
+  data?: {
+    source: string;
+    models: string; // JSON string of CatalogModel[]
+  };
+};
```

Comment:
You know what? I'm sorry - I think we can just delete this whole type. It's not used anywhere - I was about to suggest how we can use a type guard to have the value of `getConfigMap` use this type below, but the only thing that would do for us is protect against typos in `configMap.data?.modelCatalogSource` which you write on two lines - not worth the effort for a temporary workaround. Let's ditch this type.